tests/Unit/Traits/HasGuidInLdapTest.php

```diff
@@ -2,15 +2,14 @@
 
 namespace NetworkRailBusinessSystems\UserLogin\Tests\Unit\Traits;
 
-use Mockery as ldapMocker;
 use NetworkRailBusinessSystems\UserLogin\Tests\TestCase;
-use NetworkRailBusinessSystems\UserLogin\Traits\LdapUniqueIdentifier;
+use NetworkRailBusinessSystems\UserLogin\Traits\HasGuidInLdap;
 
-class LdapUniqueIdentifierTest extends TestCase
+class HasGuidInLdapTest extends TestCase
 {
     public function testRetrieveUniqueIdentifierFromLdap()
     {
-        $ldapModelMock = ldapMocker::mock('alias:LdapRecord\Models\ActiveDirectory\User');
+        $ldapModelMock = $this->mock('alias:LdapRecord\Models\ActiveDirectory\User');
```

Comment:
You can use the second parameter of `$this-mock()` to provide the mock details.

```php
$this->mock(..., function ($mock) {
    $mock->shouldReceive...
});
```

Reply:
fixed